### PR TITLE
Update polkadot to v0.9.27

### PR DESCRIPTION
**Pull request by Niederb**

### CODE REVIEW

File: claims/Cargo.toml [hunk 1]
```diff
@@ -11,21 +11,21 @@ scale-info = { version = "2.0.1", default-features = false, features = ["derive"
 serde = { version = "1.0.13", default-features = false }
 serde_derive = { version = "1.0.13", optional = true }
 
-sp-std = { package = "sp-std", default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
-sp-io = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
-sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
-frame-support = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
-frame-system = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
-frame-benchmarking = { optional = true, default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
+sp-std = { package = "sp-std", default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.27" }
```
Comment (gaudenzkessler) [inline claims/Cargo.toml:14]: can you sort these lines alphabetically?

File: claims/Cargo.toml [hunk 2]
```diff
@@ -11,21 +11,21 @@ scale-info = { version = "2.0.1", default-features = false, features = ["derive"
...
+frame-system = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.27" }
+frame-benchmarking = { optional = true, default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.27" }
 libsecp256k1 = { version = "0.7.0", default-features = false, optional = true }
 claims-primitives = {package = "claims-primitives", path = "../primitives/claims", default-features = false}
 
 [dev-dependencies]
 hex-literal = "0.3.3"
-sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
-pallet-balances = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.27" }
```
Comment (gaudenzkessler) [inline claims/Cargo.toml:25]: can you sort these lines alphabetically?

File: primitives/claims/Cargo.toml [hunk 3]
```diff
@@ -13,9 +13,9 @@ parity-scale-codec = { version = "3.0.0", default-features = false, features = [
 rustc-hex = { version = "2.1.0", default-features = false }
 serde = { version = "1.0.13", default-features = false }
 
-sp-std = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
-sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
-sp-io = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.26" }
+sp-std = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.27" }
```
Comment (gaudenzkessler) [inline primitives/claims/Cargo.toml:16]: can you sort these lines alphabetically?

**Review by gaudenzkessler — APPROVED** [review]
Very nice

**Review by clangenb — APPROVED** [review]
Nice

**Pull request merged**